File: asn1jrt/src/main/java/org/asn1gen/runtime/java/BadEnumerationException.java
package org.asn1gen.runtime.java;

@SuppressWarnings("serial")
public class BadEnumerationException extends RuntimeException {
  public BadEnumerationException() {
  }
  
  public BadEnumerationException(final String message) {
    super(message);
  }

  public BadEnumerationException(final String message, final Throwable cause) {
    super(message, cause);
  }

  public BadEnumerationException(final Throwable cause) {
    super(cause);
  }
}

File: asn1runtime/src/main/scala/org/asn1gen/runtime/BadEnumerationException.scala
package org.asn1gen.runtime

class BadEnumerationException(reason: String, cause: Throwable) extends AsnException(reason, cause) {
}

object BadEnumerationException {
  def apply(reason: String, cause: Throwable) = new BadEnumerationException(reason, cause)
  
  def apply(reason: String) = new BadEnumerationException(reason, null)
}
